edwin/client/static/js/stores/BugStore.js

```diff
@@ -88,6 +88,18 @@ function getBugState(bug) {
   return BugStates.NOT_READY;
 }
 
+function getNeedinfo(bug) {
+  // Filters the flags for needinfo flags.
+  return bug.get('flags', [])
+    .filter(flag => flag.get('name', '') === 'needinfo')
+    .map(function (flag) {
```

Comment:
This should probably use a es6-style function


```js
.map(flag => ({name: flag.get('requestee', 'Unknown'}))
```

or a slightly less terse version

```js
.map(flag => {
  return {
    name: flag.get('requestee', 'Unknown'),
  };
})
```